src/grt/.gitignore

```diff
@@ -11,5 +11,6 @@ test/src/*/out.guide
 test/src/*/out2.guide
 test/src/*/route.guide
 test/results
+test/failures_report.json
```

Comment:
What creates this?  I'm wondering if other modules would be the same and we should put it at the OR level.